platform/o.n.bootstrap/launcher/windows/platformlauncher.cpp

```diff
@@ -662,6 +664,10 @@ bool PlatformLauncher::restartRequested() {
 
 void PlatformLauncher::onExit() {
     logMsg("onExit()");
+    if (exitStatus == -252) {
```

Comment:
OK.

Reply:
How does it work with `nbexec*.exe` or `nbexec*.dll`? Do the binaries also need to be uploaded somewhere?

Reply:
! Yes, need to be uploaded to osuosl !